app/controllers/api/v1/stats/channels_controller.rb

```diff
@@ -0,0 +1,62 @@
+class Api::V1::Stats::ChannelsController < Api::BaseController
+
+  def index
+
```

Comment:
Typically Rails won't have these controllers with new lines in the beginning and end.  I think we should introduce some linting rules to prevent this? That's probably something that i'll add when I join the team 😄

Reply:
Added ✔️ , that sounds good.